README.md

```diff
@@ -108,6 +108,7 @@ module "secured_data_warehouse" {
 | enable\_bigquery\_read\_roles\_in\_data\_ingestion | (Optional) If set to true, it will grant to the dataflow controller service account created in the data ingestion project the necessary roles to read from a bigquery table. | `bool` | `false` | no |
 | key\_rotation\_period\_seconds | Rotation period for keys. The default value is 30 days. | `string` | `"2592000s"` | no |
 | kms\_key\_protection\_level | The protection level to use when creating a key. Possible values: ["SOFTWARE", "HSM"] | `string` | `"HSM"` | no |
+| labels | (Optional) Default label used by Data Warehouse resources. | `map(string)` | `{}` | no |
```

Comment:
Odd. I thought you fixed the wording already per
https://github.com/GoogleCloudPlatform/terraform-google-secured-data-warehouse/pull/282#discussion_r822272509

Reply:
I might missed something. We should be good now.

Reply:
all good now